docs/development/local-development.md

```diff
@@ -132,8 +132,8 @@ You are better off to instead export the Environment Variable `RENOVATE_TOKEN` f
 
 ### Run against a real repo
 
-To make sure everything is working, create a test repo in your account, e.g. like `https://github.com/r4harry/testrepo1`.
-Now, add a file called `.nvmrc` with the content `8.13.0`.
+To make sure everything is working, create a test repository in your account, e.g. like `https://github.com/r4harry/testrepo1`.
+Now, add a file called `.nvmrc` with the content `20.12.2`.
```

Comment:
The `20.12.2` version is the current version in our own `.npmrc` file. So we should probably use that number?

https://github.com/renovatebot/renovate/blob/e3351ba4c64d3169b783471245a5e5ce06d28feb/.nvmrc#L1C1-L1C8

Reply:
The reason for nvmrc was to have a very simple package file example. It doesn't need to be the latest

Reply:
20.0.0 can be sufficient

Reply:
```suggestion
Now, add a file called `.nvmrc` with the content `20.0.0`.
```